### ISSUE TITLE
confd from calico/node v3.0.4 still use api v1  to access etcd3

### ISSUE THREAD

**Issue by likunbyl** [projectcalico/calico#1803]
Following is excerpt from the template file /etc/calico/confd/templates/bird.cfg.template: 

```
# ------------- Global peers -------------
{{if ls "/global/peer_v4"}}
{{range gets "/global/peer_v4/*"}}{{$data := json .Value}}
{{$nums := split $data.ip "."}}{{$id := join $nums "_"}}
```

It's obviously using the api v1 to get values from etcd.
My confd version:
```
# confd -version
confd v1.0.3
```

It's supposed to using the api v3 to access etcd, am I missing something here?

**Comment by caseydavenport** [comment]
@likunbyl it's pretty confusing, but it is actually doing the right thing.

The values in the templates are in v1 format, but confd has the knowledge to populate them with v3 data. The confd version is actually correct - v1.0.3 is the latest version of confd.

Right now, our version numbers don't all align. I hope to fix that in v3.1 (make all components use a v3.1.x version for consistency).

**Comment by caseydavenport** [comment]
here is the code in confd which loads the Calico v3 client: https://github.com/projectcalico/confd/blob/master/backends/calico/client.go#L60
